components/trace/appmap/classmap/exclusion.mjs

```diff
@@ -21,25 +22,25 @@ export default (dependencies) => {
       if (parent.type === "class") {
         return `${parent.name}${entity.static ? "#" : "."}${entity.name}`;
       }
-      assert(false, "getName called on invalid parent entity");
+      throw assert(false, "getName called on invalid parent entity");
     }
-    assert(false, "getName called on invalid entity");
+    throw assert(false, "getName called on invalid entity");
```

Comment:
Does `assert` not raise an exception? Why not just use the following?

```js
throw new Error('getName called on invalid entity');
```

Reply:
I hesitated to switch to an explicit error. But I agree this is less confusing. However this will not work for `expect` which is used for signaling user mistakes.

Reply:
done 62be2a5f798cfff26eb8cf070f675e2f3c29cec9